docs/general/how-ra-works.md

```diff
@@ -0,0 +1,45 @@
+# How RetroAchievements Works
+
```

Comment:
```
# How RetroAchievements Works

[[toc]]

... content here ...
```

Is adding a table of contents near the top of the page desirable?

Reply:
I think it is.  Added per suggestion.